lib/galaxy/datatypes/neo4j.py

```diff
@@ -99,36 +121,73 @@ def __init__(self, **kwd):
         self.add_composite_file('neostore.counts.db.a', is_binary=True)
         self.add_composite_file('neostore.counts.db.b', is_binary=True)
```

Comment:
Can we make this `optional=True` because I've seen Neo4j datastores where this file is missing.